### ISSUE TITLE
Request for Feedback: "Scaling Cloud Controller" guidance document

### ISSUE THREAD

**Issue by chenl23** [chenl23/cfar-proposals#12]
### Proposal Name
Request for Feedback: "Scaling Cloud Controller" guidance document

### TL;DR Summary
The CAPI team has been working on a document to give operators some guidance with regards to knowing when and how to scale their Cloud Controllers (and related jobs in capi-release) based on our team's experience. 

A draft version of the "Scaling Cloud Controller" document can be viewed here:
https://github.com/cloudfoundry/capi-release/blob/develop/docs/scaling_cloud_controller.md

Our intent is to work with the Docs team and get this added as part of the Running Cloud Foundry operator docs, but first we'd welcome and appreciate any feedback from the CF operator community. Feel free to respond to this post or submit an issue/PR with suggestions for the draft document.

### Proposal URL
https://docs.cloudfoundry.org/running/managing-cf/scaling-cloud-controller.html

### Point of Contact
Tim Downey

**Comment by tcdowney** [comment]
@chenl23 this request to the CF Community for feedback on our Cloud Controller scaling guidance doc should be considered closed since the guidelines are now live.

Any additional feedback/corrections should be made as Github Issues or PRs to the following docs repo:

https://github.com/cloudfoundry/docs-running-cf/blob/master/managing-cf/scaling-cloud-controller.html.md.erb

**Comment by chenl23** [comment]
Updated issue to complete status.
